paymentsheet/src/test/java/com/stripe/android/paymentsheet/forms/FormViewModelTest.kt

```diff
@@ -103,20 +133,25 @@ class FormViewModelTest {
 
             val saveForFutureUseController = formViewModel.elements.map { it.controller }
                 .filterIsInstance(SaveForFutureUseController::class.java).first()
-            val emailController = formViewModel.elements.map { it.controller }
+            val emailController = formViewModel.elements
+                .asSequence()
+                .filterIsInstance<SectionElement>()
+                .map { it.fields }
+                .flatten()
+                .map { it.controller }
                 .filterIsInstance(TextFieldController::class.java).first()
 
```

Comment:
I think this can be simplified to

```
            val emailController = formViewModel.elements
                .filterIsInstance<SectionElement>()
                .flatMap { it.fields }
                .map { it.controller }
                .filterIsInstance<TextFieldController>()
                .first()
```

Reply:
👍